src/librustc_mir/monomorphize/collector.rs

```diff
@@ -25,21 +25,7 @@
 //! The following kinds of "translation items" are handled here:
 //!
 //! - Functions
-//! - Methods
-//! - Closures
-//! - Statics
-//! - Drop glue
-//!
-//! The following things also result in LLVM artifacts, but are not collected
-//! here, since we instantiate them locally on demand when needed in a given
-//! codegen unit:
-//!
-//! - Constants
-//! - Vtables
-//! - Object Shims
-//!
-//!
-//! General Algorithm
+//! - Methm
```

Comment:
This looks to be accidentally truncated.

Reply:
Not sure what happened, I am fixing it now.